code/API_definitions/geofencing.yaml

```diff
@@ -0,0 +1,658 @@
+          description: The list of subscriptions is retrieved.
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: '#/components/schemas/SubscriptionInfo'
+        '400':
+          $ref: '#/components/responses/Generic400'
+        '401':
+          $ref: '#/components/responses/Generic401'
+        '403':
+          $ref: '#/components/responses/Generic403'
+        '500':
+          $ref: '#/components/responses/Generic500'
+        '503':
+          $ref: '#/components/responses/Generic503'
+      security:
+        - oAuth2AuthorizationCode:
+            - "geofencing:subscriptions:read"
+
+  /subscriptions/{subscriptionsId}:
+    get:
+      tags:
+        - Geofencing
+      summary: 'Operation to retrieve an subscription'
+      operationId: getSubscription
+      description: Retrieve a given subscription by ID
+      parameters:
+        - name: subscriptionsId
+          in: path
+          description: subscription identifier that was obtained from the create event subscription operation
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Contains information about Subscriptions
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SubscriptionInfo'
+        '400':
+          $ref: '#/components/responses/Generic400'
+        '401':
+          $ref: '#/components/responses/Generic401'
+        '403':
+          $ref: '#/components/responses/Generic403'
+        '404':
+          $ref: '#/components/responses/Generic404'
+        '500':
+          $ref: '#/components/responses/Generic500'
+        '503':
+          $ref: '#/components/responses/Generic503'
+      security:
+        - oAuth2AuthorizationCode:
+            - "geofencing:subscriptions:read"
+    delete:
+      tags:
+        - Geofencing
+      summary: 'Operation to delete a subscription'
+      operationId: deleteSubscription
+      description: Delete a given subscription by ID
+      parameters:
+        - name: subscriptionsId
+          in: path
+          description: subscription identifier
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Event subscription deleted
+        '400':
+          $ref: '#/components/responses/Generic400'
+        '401':
+          $ref: '#/components/responses/Generic401'
+        '403':
+          $ref: '#/components/responses/Generic403'
+        '404':
+          $ref: '#/components/responses/Generic404'
+        '500':
+          $ref: '#/components/responses/Generic500'
+        '503':
+          $ref: '#/components/responses/Generic503'
+      security:
+        - oAuth2AuthorizationCode:
+            - "geofencing:subscriptions:write"
+
+components:
+  securitySchemes:
+    oAuth2ClientCredentials:
```

Comment:
This is not used in practice

Reply:
So I can delete oAuth2ClientCredentials - part here in the "securitySchemes"?